builtin/credential/aws/path_role.go

```diff
@@ -436,7 +499,9 @@ func (b *backend) pathRoleCreateUpdate(ctx context.Context, req *logical.Request
 		return nil, err
 	}
 	if roleEntry == nil {
-		roleEntry = &awsRoleEntry{}
+		roleEntry = &awsRoleEntry{
+			version: currentRoleStorageVersion,
```

Comment:
This value can't be unexported, because JSON can't marshal/unmarshal if it is. So it will end up always being 0 on read.

Reply:
Good call out. I didn't want to return this to clients, and that was before I stole the ToResponseData method, so I've now exported it.